runelite-client/src/main/java/net/runelite/client/plugins/cleanermenus/CleanerMenusPlugin.java

```diff
@@ -0,0 +1,57 @@
+package net.runelite.client.plugins.cleanermenus;
+
+import com.google.common.eventbus.Subscribe;
+import net.runelite.api.Client;
+import net.runelite.api.MenuEntry;
+import net.runelite.api.events.MenuOpened;
+import net.runelite.client.plugins.Plugin;
+import net.runelite.client.plugins.PluginDescriptor;
+
+import javax.inject.Inject;
+import java.util.AbstractMap;
+import java.util.HashMap;
+
+@PluginDescriptor(name = "Cleaner Menus")
```

Comment:
Can this plugin be disabled by default? This is behavior is very "different" from what's expected from RuneScape. Similar to the Menu Entry Swapper.

Reply:
Hmm, just read the comments. If this gets merged, the option should be disabled by default IMO.